### ISSUE TITLE
Clean up database views

### ISSUE THREAD

**Issue by raif-s-naffah** [raif-s-naffah/xapi-rs#1]
When reacting to **`GET`** `/statements` resource requests resulting in multiple _statements_, we construct a `StatementResult` structure that contains two properties:

* `statements`: a collection of `Statement` objects, and
* `more`: an IRL pointing back to us if not all the filtered _statements_ were included in the previous property.

The number of `Statement` objects in `statements` is limited by the configuration parameter `DB_STATEMENTS_PAGE_LEN`.

In order to honour the current **`GET`** request and potentially following ones targeting the `IRL` we provided in `more`, we create one or more database _Views_ each addressing certain criteria constructed from the filter query parameters.

All those views share a name prefix constructed from catenating the character `v` w/ a random number. For example, running the following SQL
```sql
SELECT viewname FROM pg_views WHERE schemaname = 'public' ORDER BY viewname
```
may result in the following:
| viewname      |
| ------------- |
| v10033724414410780193 |
| v10033724414410780193a |
| v10033724414410780193e |
| ...|
| v10465880322953843163 |
|v10465880322953843163d|

Those _Views_ need to be removed after a certain time.

**Comment by raif-s-naffah** [comment]
I'll be checking in soon a fix for this.  Here's how it will work...

* In the [lrs/resources/statement#get_many][1], rather than using a random number as the`sid`, we get a (new) table row ID which records the timestamp (in seconds) of when a reservation was made.
* This new table, called `filter`, has 2 columns: `id` and `created`.  Both will be populated from default values.  A new _migration_ file will be added to create said table.
* In the [Fairing implementation][2] of `DB` i'll add an `on_liftoff` and `on_shutdown` hooks to (a) periodically run a task to find and delete stale `sid`s and related views, while LaRS is up, and (b) do the same for all records/`sid`s still in `filter` at shutdown.
* Three new configuration parameters, will be added to `.env.template` to control:
  - Time-To-Live of a `sid` and its related Views,
  - How many stale `sid` rows (from `filter`) to process at a time, and
  - How long the scheduler should sleep between 2 consecutive runs. 

[1]: https://github.com/raif-s-naffah/xapi-rs/blob/8424344ec5ad4cfd4a6942a0b8496098d20763e0/src/lrs/resources/statement.rs#L1262
[2]: https://github.com/raif-s-naffah/xapi-rs/blob/8424344ec5ad4cfd4a6942a0b8496098d20763e0/src/lrs/db.rs#L87

**Comment by raif-s-naffah** [comment]
fixed as of release 0.1.1.
